src/uneditableTextField/index.unit.tsx

```diff
@@ -1,41 +1,74 @@
 import React from 'react'
-import renderer from 'react-test-renderer'
 
-import UneditableTextField from './index'
+import { render, screen } from '@testing-library/react'
+
+import UneditableTextField, { UneditableTextFieldProps } from './index'
+
+function createProps(props: Partial<UneditableTextFieldProps> = {}): UneditableTextFieldProps {
+  return { children: 'Hello world', ...props }
+}
 
 describe('UneditableTextField', () => {
-  it('Should have the proper default props', () => {
-    const wrapper = renderer.create(<UneditableTextField>Hello world</UneditableTextField>)
-    expect(wrapper.toJSON()).toMatchSnapshot()
+  it('should render the content', () => {
+    const props = createProps()
+    const { container } = render(<UneditableTextField {...props} />)
+
+    expect(container.firstChild).toHaveTextContent(props.children as string)
+  })
+
+  it('should apply public CSS classes', () => {
```

Comment:
I not sure we should have CSS classes in the public API of a component but they are used so I added a specific test to be isofunctional.